inference-engine/src/mkldnn_plugin/emitters/jit_eltwise_emitters.cpp

```diff
@@ -1224,16 +1285,44 @@ size_t jit_logical_not_emitter::aux_vecs_count() const {
 }
 
 /// POWER_STATIC ///
+jit_power_static_emitter::jit_power_static_emitter(jit_generator *host, cpu_isa_t host_isa, const std::shared_ptr<ngraph::Node>& node, Precision exec_prc)
+: jit_emitter(host, host_isa, node, exec_prc) {
+    auto parent = node->input(1).get_source_output().get_node_shared_ptr();
+    if (!std::dynamic_pointer_cast<ngraph::op::Constant>(parent)) {
+        throw ngraph::ngraph_error("unsupported non constant power");
+    }
+
+    if (!(node->input(1).get_shape() == ngraph::Shape() || ngraph::shape_size(node->input(1).get_shape()) == 1)) {
+        throw ngraph::ngraph_error("unsupported non scalar power");
+    }
+    power = ngraph::as_type_ptr<ngraph::op::Constant>(parent)->get_data_ptr<float>()[0];
+    scale = 1.f;
+    shift = 0.f;
+    push_arg_entry_of("power", float2int(power), true);
```

Comment:
seems like these lines duplicate the logic inside `jit_power_static_emitter::register_table_entries`